Subject: Bug fix - missing break for the loop by possible main function names.

If main function name found then end the loop.

>From : Anatoli Koutsevol <Anatoli.Koutsevol@MarkovProcesses.com>

Index: frontend/frontend.cpp
===================================================================
RCS file: /cvsroot/boomerang/boomerang/frontend/frontend.cpp,v
retrieving revision 1.49
diff -u -r1.49 frontend.cpp
--- frontend/frontend.cpp	1 Feb 2004 04:59:30 -0000	1.49
+++ frontend/frontend.cpp	1 Feb 2004 18:41:20 -0000
@@ -185,6 +185,7 @@
                 assert(fty);
                 proc->setSignature(fty->getSignature()->clone());
                 proc->getSignature()->setName(name);
+                break;
             }
         }
     }
